client/blocks/comments/comment-actions.scss

```diff
@@ -7,6 +7,7 @@
 	gap: 22px;
 	justify-content: flex-start;
 	list-style: none;
+	margin-top: 4px;
```

Comment:
We were implicitly getting extra height from line height in the action items (which was in part causing the alignment issue). After resetting line height back to a baseline, the comment actions felt too tight against the comment, so this adds back that space explictly.